app/models/speedrun_dot_com_game.rb

```diff
@@ -0,0 +1,28 @@
+require 'speedrundotcom'
+
+class SpeedrunDotComGame < ApplicationRecord
+  belongs_to :game
+
+  def self.from_game!(game)
+    return game.srdc if game.srdc.present?
+
+    result = SpeedrunDotCom::Game.search(game.name).first
+    new(srdc_id: result['id'], game: game).from_result!(result)
+  end
+
+  def sync!
+    from_result!(SpeedrunDotCom::Game.from_id(srdc_id))
+  end
+
+  def from_result!(result)
+    update(
+      name:           result['names']['international'],
```

Comment:
It might be worth storing the twitch name as well if you ever plan to add more direct links to things on twitch.  I'm not sure if it's present in the docs but for things like Yoshi's Island its there.

https://www.speedrun.com/api/v1/games/nd280rd0